### ISSUE TITLE
Please tag the latest version

### ISSUE THREAD

**Issue by mikesoule** [renatomefi/php-fpm-healthcheck#59]
Just posting to request a tag of the latest version in `master`.

**Comment by smulvih2** [comment]
Yes, please tag the latest master branch, the last release is 28 commits behind.

**Comment by renatomefi** [comment]
Hey, sorry I'm not maintaining anymore. I've released the tag.

If anyone wants to take over the project let me know. cheers
